apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+import ReactSelect, {
+	type GroupBase,
+	type OptionsOrGroups,
+} from "react-select";
+
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useFormController } from "~/libs/hooks/hooks.js";
+
+import { type Option } from "./select.js";
+import styles from "./styles.module.css";
+
+type Properties<TFieldValues extends FieldValues, TValue> = {
+	control: Control<TFieldValues, null>;
+	isMulti?: boolean;
+	name: FieldPath<TFieldValues>;
+	options:
+		| OptionsOrGroups<Option<TValue>, GroupBase<Option<TValue>>>
+		| undefined;
+	title?: string;
```

Comment:
rename it to label and it should be required
```suggestion
	label: string;
```